app/controllers/pets_controller.rb

```diff
@@ -9,6 +9,11 @@ def index
 
   def show
     @pet = Pet.find(params[:id])
+    if @pet.any_approved?(@pet.id)
+      @pet.update!(adoptable: false)
+    else
+    
```

Comment:
if the else part of this statement is empty you can just have an if statement. No need for the else.